app/scripts/lib/get-first-preferred-lang-code.js

```diff
@@ -2,8 +2,7 @@ const extension = require('extensionizer')
 const promisify = require('pify')
 const allLocales = require('../../_locales/index.json')
 
-const isSupported = extension.i18n && extension.i18n.getAcceptLanguages
-const getPreferredLocales = isSupported ? promisify(
+const getPreferredLocales = extension.i18n ? promisify(
```

Comment:
My understanding is that Brave _does_ expose a `extension.i18n` object, and even a `getAcceptLanguages` method on it, so they would pass this test.

From what they reported and I relayed [in this comment](https://github.com/MetaMask/metamask-extension/issues/4270#issuecomment-395154557) is that the only way to check their failure right now is to check `chrome.runtime.lastError` within the callback, and if there's an error, _then_ fall back.

Reply:
Per feedback from Brave, we are now handling errors from this function, because Brave currently throws for this.